contracts/src/arbitration/IArbitrator.sol

```diff
@@ -0,0 +1,98 @@
+        Appealable,
+        Solved
+    }
+
+    /**
+     * @dev To be emitted when a dispute is created.
+     * @param _disputeID ID of the dispute.
+     * @param _arbitrable The contract which created the dispute.
+     */
+    event DisputeCreation(uint256 indexed _disputeID, IArbitrable indexed _arbitrable);
+
+    /**
+     * @dev To be emitted when a dispute can be appealed.
+     * @param _disputeID ID of the dispute.
+     * @param _arbitrable The contract which created the dispute.
+     */
+    event AppealPossible(uint256 indexed _disputeID, IArbitrable indexed _arbitrable);
+
+    /**
+     * @dev To be emitted when the current ruling is appealed.
+     * @param _disputeID ID of the dispute.
+     * @param _arbitrable The contract which created the dispute.
+     */
+    event AppealDecision(uint256 indexed _disputeID, IArbitrable indexed _arbitrable);
+
+    /**
+     * @dev Create a dispute. Must be called by the arbitrable contract.
+     * Must pay at least arbitrationCost(_extraData).
+     * @param _choices Amount of choices the arbitrator can make in this dispute.
+     * @param _extraData Can be used to give additional info on the dispute to be created.
+     * @return disputeID ID of the dispute created.
+     */
+    function createDispute(uint256 _choices, bytes calldata _extraData) external payable returns (uint256 disputeID);
+
+    /**
+     * @dev Compute the cost of arbitration. It is recommended not to increase it often, as it can be highly time and gas consuming for the arbitrated contracts to cope with fee augmentation.
+     * @param _extraData Can be used to give additional info on the dispute to be created.
+     * @return cost Required cost of arbitration.
+     */
+    function arbitrationCost(bytes memory _extraData) external view returns (uint256 cost);
+
+    /**
+     * @dev Compute the cost of an appeal.
+     * @param _disputeID ID of the dispute to be appealed.
+     * @param _extraData Can be used to give additional info of the dispute.
+     * @return cost Required appeal cost.
+     */
+    function appealCost(uint256 _disputeID, bytes memory _extraData) external view returns (uint256 cost);
+
+    /**
+     * @dev Compute the start and end of the dispute's appeal period, if possible. If appeal is impossible: should return (0, 0).
+     * @param _disputeID ID of the dispute.
+     * @return start The start of the period.
+     * @return end The end of the period.
+     */
+    function appealPeriod(uint256 _disputeID) external view returns (uint256 start, uint256 end);
+
+    /**
+     * @dev Make a contribution to fund one (or the subset) of possible choices in order to appeal the ruling.
+     * Note that the desired appeal system will be defined by the selected Dispute Kit.
+     * @param _disputeID The ID of the dispute to appeal.
+     * @param _choices The choices to contribute to.
+     */
+    function fundAppeal(uint256 _disputeID, uint256[] calldata _choices) external payable;
+
```

Comment:
A view function showing the funding status, funding goal and number of required funded choices would be useful. It's different from the old `appealCost()` which did not account for the funders reward. Maybe something like:
```solidity
    /**
     * @dev Return the funded amount and funding goal for this choice, as well as the minimum number of choices which must be funded to obtain an appeal. The status info may not be available if the dispute is not currently appealable.
     * @param _disputeID The ID of the dispute to appeal.
     * @param _choice The choice corresponding to this status.
     * @return funded The amount funded so far for this choice in wei.
     * @return goal The amount to fully fund this choice in wei.
     * @return minNbOfChoicesToFund The number of choices which must be fully funded to obtain an appeal.
     */
    function fundingStatus(uint256 _disputeID, uint256 _choice)
        external
        view
        returns (
            uint256 funded,
            uint256 goal,
            uint256 minNbOfChoicesToFund
        );
```

Reply:
See my comment in the doc. 
uint256 minNbOfChoicesToFund - Is it possible to have more than two choices though? Even if fund a certain subset it'll be always funded as a countermeasure to the first funded choices that should always be separated, according to the yellow paper. So even with the subset funded choices are de-facto just 2

Reply:
Indeed that's more consistent: `uint256 _choice` --> `uint256[] _choices`